type_specs/query/ai_query.json

```diff
@@ -174,6 +174,9 @@
         "ListStores": "UNIT"
       },
       "10": {
+        "PurgeStores": "UNIT"
```

Comment:
Could you modify the Doc [here](https://github.com/deven96/ahnlich/blob/main/docs/ai.md) to reflect all ai queries accurately